### PR TITLE
app passwords/device tokens wording

### PR DESCRIPTION
**Pull request by ChristophWurst**
see https://github.com/owncloud/client/pull/4963#issuecomment-226419705
- [x] TODO: preserve anchor on redirects

@ogoffart @PVince81 @guruz as discussed

### CODE REVIEW

File: settings/templates/personal.php [hunk 1]
```diff
@@ -169,14 +169,14 @@
 		<tbody class="token-list icon-loading">
 		</tbody>
 	</table>
-	<p><?php p($l->t('A device password is a passcode that gives an app or device permissions to access your ownCloud account.'));?></p>
-	<div id="device-token-form">
-		<input id="device-token-name" type="text" placeholder="Device name">
-		<button id="device-add-token" class="button">Create new device password</button>
+	<p><?php p($l->t('An app password is a passcode that gives an app or device permissions to access your ownCloud account.'));?></p>
```
Comment (DeepDiver1975) [inline settings/templates/personal.php]: ownCloud needs to be replaced with $theme->getName() for theming reasons

Comment (ChristophWurst) [inline settings/templates/personal.php]: done

**Comment by PVince81** [comment]
@owncloud/designers for more feedback about the wording

**Comment by guruz** [comment]
👍

**Comment by PVince81** [comment]
Now I hope people won't think "App" is a PHP ownCloud app.

**Comment by MorrisJobke** [comment]
> Now I hope people won't think "App" is a PHP ownCloud app.

Oh. Good point. But the vast majority of users usually don't see the app reference (as ownCloud app, app management, ...) in the ownCloud web UI. They only notice that there are different functions. Which then means that it is not that bad at all.

**Comment by ChristophWurst** [comment]
@ogoffart FYI this will change the device token url to ``/settings/personal?section=apppasswords``

**Comment by PVince81** [comment]
:+1:

Also tested the section attr and worked fine

**Comment by PVince81** [comment]
@guruz @dragotin please review before merging https://github.com/owncloud/client/pull/4963 :smile:

**Comment by guruz** [comment]
👍

**Pull request merged**